price-feeder/oracle/provider/kraken_test.go

```diff
@@ -71,7 +71,7 @@ func TestKrakenProvider_GetTickerPrices(t *testing.T) {
 	t.Run("invalid_request_invalid_ticker", func(t *testing.T) {
 		prices, err := p.GetTickerPrices(types.CurrencyPair{Base: "FOO", Quote: "BAR"})
 		require.Error(t, err)
-		require.Equal(t, "failed to get ticker price for FOOBAR", err.Error())
+		require.Equal(t, "kraken failed to get ticker price for FOOBAR", err.Error())
```

Comment:
instead of 2 above assertions you could use
+ https://pkg.go.dev/github.com/stretchr/testify/assert#Assertions.EqualError
+ or https://pkg.go.dev/github.com/stretchr/testify/assert#Assertions.ErrorContains